CHANGELOG.md

```diff
@@ -1,74 +1,83 @@
+- `tPrint` and reconcile printing options
+- Python unit test coverage report
+- Factor embed library logic into function and test
+- `clang++` as cxx-compiler option for Windows
+- Logic to cope with different compilers
+- `generateManifest` function and rename generateManifest to toFile and move to Utilities
+- Profiling CI job
+- Support for `amdclang` and use defaults
+- Architecture management functions in `TensileCreateLibrary`
+- `TensileCreateLibrary` CLI reference docs
+- New documentation for sphinx prototype and build out skeleton
+- Prediction model for optimal number of Stream-K tiles to run
+- Two-tile algorithm with Stream-K after DP
+- Atomic two-tile Stream-K and clean-up tuning parameters
 
-### Changes
-- improve rocBLAS build output by allowing warning suppression, ignoring only developer warnings, progress bar and quiet printing
-- reorder extensions for Windows in which function
-- remove deprecated flag from CI profiling job
-- update amdclang++ and asm directories
-- update duplicate marking tests with mocks
-- remove diagnostic print, and restore print ordering, and add missing print option
-- bump rocm-docs-core from 1.2.0 to 1.5.0 in /docs/sphinx
-- refactor kernel duplicate matching
-- refactor generateLogicDataAndSolutions
-- remove globals from prepAsm
-- restrict XCC mapping to gfx942
-- refactor argument parsing in TensileCreateLibrary
-- disable failing rhel9 tests
-- change line length for formatting to 100 characters
-- change YAML operations to use C libyaml backend
-- improve warning wording
-- remove deprecated package-library option
-- update clang support for Windows
-- update supportedCompiler fxn
+### Changed
+
+- Improved rocBLAS build output by allowing warning suppression, ignoring developer warnings, displaying progress bar and quiet printing
+- Reordered extensions for Windows in which function
+- updated `amdclang++` and `asm` directories
+- Updated duplicate marking tests with mocks
+- Restored print ordering
+- Print option
+- Bumped rocm-docs-core from 1.2.0 to 1.5.0 in `/docs/sphinx`
+- Refactored kernel duplicate matching
+- Refactored `generateLogicDataAndSolutions`
+- Restricted XCC mapping to gfx942
+- Refactored argument parsing in `TensileCreateLibrary`
+- Disabled failing rhel9 tests
+- Changed line length to 100 characters for formatting
+- Changed YAML operations to use C `libyaml` backend
+- Improved warning text
+- Updated clang support for Windows
+- Updated `supportedCompiler` function
 - use conditional choices and defaults
-- remove duplicate which function and minor cleanup
-- refactor sanity check in TensileCreateLibrary
-- factor client config logic from TensileCreateLibrary main into createClientConfig
+- Refactored sanity check in `TensileCreateLibrary`
+- Moved client config logic from `TensileCreateLibrary` main into `createClientConfig`
 - use glob to find logic files in TensileCreateLibrary
 - use function to confirm supported compiler rather than raw logic
```

Comment:
@bstefanuk What exactly is changing here? It seems like this should be listed under "Added".

Reply:
A function was implemented to encapsulate logic.

Sometimes the difference between a change and addition is fuzzy, you can move this into "Added" if you please.